account_invoice_mgmt_dms/models/account_move.py

```diff
@@ -119,29 +134,60 @@ def create_dms_file_document_manager(self):
             if dms_file_carrier_ids:
                 for doc_carrier in dms_file_carrier_ids:
                     streams.append(io.BytesIO(doc_carrier.content_binary))
+                    if validate:
+                        error += self.validation_files(doc_carrier)
 
         if not proceeding:
             proceeding = self.name
         dms_extra_ids = self.env['dms.file'].sudo().search([('proceeding', 'ilike', proceeding), ('directory_id', '=', self.env.ref('account_invoice_mgmt_dms.dms_directory_extra_doc').id)])
         if dms_extra_ids:
             for doc_extra in dms_extra_ids:
                 streams.append(io.BytesIO(doc_extra.content_binary))
+                if validate:
+                    error += self.validation_files(doc_extra)
 
         if self.complete_proceesing_id:
             self.complete_proceesing_id.unlink()
 
+        if error:
+            raise ValidationError(_("Error in files: %s \n Must uploaded again") % error)
+        
         attachment = self.env['ir.actions.report']._merge_pdfs(streams)
 
+        # Save the PDF to a temporary file
+        with tempfile.NamedTemporaryFile(suffix='.pdf', delete=False) as temp_pdf:
+            temp_pdf.write(attachment)
+            temp_pdf_path = temp_pdf.name
+
+        # Compress PDF using Ghostscript
+        compressed_pdf_path = temp_pdf_path.replace('.pdf', '_compressed.pdf')
+        args = [
+            "ps2pdf",
+            "-dNOPAUSE", "-dBATCH", "-dSAFER","-dPDFSETTINGS=/ebook",
```

Comment:
Is there are link in internet for `/ebook` setting addition, that can be used as explanation?